sysl2/naive/grammarParser.go

```diff
@@ -111,7 +111,7 @@ func GetTermMinMaxCount(t *sysl.Term) (int, int) {
 
 func (p *parser) parse(g *sysl.Grammar, input int, val interface{}) (bool, int, []interface{}) {
 	if input == len(*(p.tokens)) {
-		logrus.Println("got input length zero!!!")
+		logrus.Debugln("got input length zero!!!")
```

Comment:
Use `Debug` instead of `Debugln` (for the same reason that `Debugf` format strings don't end with `\n`).